roles/edpm_neutron_sriov/tasks/install.yml

```diff
@@ -19,9 +19,11 @@
     path: "{{ item.path }}"
```

Comment:
you should have become: true here

now you are relying on the become form the playbook level which is incorrect

Reply:
> you should have become: true here
> 
> now you are relying on the become form the playbook level which is incorrect

Ack